tests/xquant_tests/keras_tests/test_xquant_report.py

```diff
@@ -0,0 +1,176 @@
+#  Copyright 2024 Sony Semiconductor Israel, Inc. All rights reserved.
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#      http://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#  ==============================================================================
+#
+
+import unittest
+from functools import partial
+import tempfile
+
+import model_compression_toolkit as mct
+from mct_quantizers import KerasQuantizationWrapper
+from xquant import XQuantConfig
+import tensorflow as tf
+
+import keras
+import numpy as np
+
+from xquant.common.constants import OUTPUT_METRICS_REPR, OUTPUT_METRICS_VAL, INTERMEDIATE_METRICS_REPR, INTERMEDIATE_METRICS_VAL
+from xquant.common.framework_report_utils import DEFAULT_METRICS_NAMES
+from xquant.keras.facade_xquant_report import xquant_report_keras_experimental
+
+
+def get_model_to_test():
+    inputs = keras.layers.Input(shape=(3, 8, 8))
+    x = keras.layers.Conv2D(3, 3)(inputs)
+    x = keras.layers.BatchNormalization()(x)
+    outputs = keras.layers.Activation('relu')(x)
+    model = keras.Model(inputs=inputs, outputs=outputs)
+    return model
+
```

Comment:
Consider adding another model that is more complicated and duplicate the tests on it.
Maybe a model that also includes a Conv without a BN after it, and without an activation, and maybe an operator (add or something).
Consider adding other common layers as well (fc, softmax...)